app/src/main/java/org/shadowice/flocke/andotp/Activities/MainActivity.java

```diff
@@ -390,7 +390,8 @@ public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
         if (key.equals(getString(R.string.settings_key_label_size)) ||
                 key.equals(getString(R.string.settings_key_label_scroll)) ||
                 key.equals(getString(R.string.settings_key_split_group_size)) ||
-                key.equals(getString(R.string.settings_key_thumbnail_size))) {
+                key.equals(getString(R.string.settings_key_thumbnail_size)) ||
+                key.equals(getString(R.string.settings_key_minimize_on_copy))) {
```

Comment:
This is not needed in this case. The MainActivity only needs to be recreated if something in the UI changes with the changed setting.
I will merge this anyways and remove it later myself to get it done a little quicker, thank you for your work,